.pre-commit-config.yaml

```diff
@@ -3,4 +3,39 @@ repos:
     rev: 0.16.0
     hooks:
     - id: check-github-actions
-    - id: check-github-workflows
+    # TODO: reactivate when fixed
+    # - id: check-github-workflows
```

Comment:
Is this TODO for this PR, or for later?

Reply:
https://github.com/getsentry/relay/pull/5070#discussion_r2303288621

\shrug